.github/workflows/test.yml

```
@@ -0,0 +1,119 @@
+
+on:
+  pull_request:
+    branches: [master, develop]
+  push:
+    branches: [master, develop]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)